plugins/terminal-proxy/lib/terminalProxy.js

```diff
@@ -391,14 +391,14 @@ TerminalWebsocketProxy.prototype.handleTerminalClientMessage = function(message,
 };
 
 TerminalWebsocketProxy.prototype.netSend = function(buffer) {
-  this.logger.debug(this.identifierString()+' Writing to host socket. Length='+buffer.length);
-  this.logger.log(this.logger.FINER,this.identifierString()+' Content to be sent to host socket=\n'+buffer);
+  this.logger.debug("ZWED0275I", this.identifierString(), buffer.length); //this.logger.debug(this.identifierString()+' Writing to host socket. Length='+buffer.length);
+  this.logger.debug("ZWED0136I", this.identifierString(), buffer); //this.logger.log(this.logger.FINER,this.identifierString()+' Content to be sent to host socket=\n'+buffer);
   this.hostSocket.write(buffer);
 };
 
 TerminalWebsocketProxy.prototype.wsSend = function(websocket,string) {
-  this.logger.debug(this.identifierString()+' Websocket sending client message. Length='+string.length);
-  this.logger.log(this.identifierString()+' Content to be sent to client=\n'+string);
+  this.logger.debug("ZWED0276I", this.identifierString(), string.length); //this.logger.debug(this.identifierString()+' Websocket sending client message. Length='+string.length);
+  this.logger.info("ZWED0137I", this.identifierString(), string); //this.logger.log(this.identifierString()+' Content to be sent to client=\n'+string);
```

Comment:
Same concern as above... you really do not want to print this by accident, it can be a lot of stuff. best to use logger.trace

Reply:
I see what happened here: .log was used without a first argument, so this was never printed before! Still, should be .trace

Reply:
Corrected to trace.